### ISSUE TITLE
M&Ms dataset

### ISSUE THREAD

**Issue by SilverBach805** [MQinghe/MiDSS#1]
Great work！ but I noticed that the source of the M&Ms dataset is not provided in the paper. Could you please provide the download link for the M&Ms dataset?

**Comment by MQinghe** [comment]
Thank you for your attention！ The M&Ms dataset is provided in the fourth cited paper, and the link is https://www.ub.edu/mnms/

**Comment by SilverBach805** [comment]
> Thank you for your attention！ The M&Ms dataset is provided in the fourth cited paper, and the link is https://www.ub.edu/mnms/
Okay, thank you.

**Comment by xumengsheng** [comment]
Great job! I've been reproducing your essay recently, and have been reading the issues you and others have discussed. When reproducing the third dataset, it was a bit off from the metrics in the third dataset of your paper, using the shared dataset, and I wanted to ask if there were any other versions of the M&M's dataset that you were training on at the time? Can you give any advice, because the labeled domain learns high dice values, but the unlabeled domain doesn't seem to be learned, and the results are generally low, do I need to go to any specific place to focus on? Also what is the difference between dice and dc within the metrics that come out of the test results, as the two values are very close. Finally, if there are other versions of the M&M dataset, could you please share the M&M dataset again, thank you very much.

**Comment by MQinghe** [comment]
Thank you for your attention to our work. Apart from the publicly available datasets, we did not use any other versions of the M&Ms dataset during training. Regarding the issue where the unlabeled domain seems not to be learned, you can log the quality of the pseudo-labels for the unlabeled samples in each domain in real-time. This would allow you to observe whether there is consistently no learning (including unlabeled samples from the same domain as the labeled ones) or if overfitting occurs in the later stages of training. Dice and DC are the same metric; Dice is defined separately, while DC is calculated using the medpy package. You can simply refer to the DC results.

**Comment by xumengsheng** [comment]
> Thank you for your attention to our work. Apart from the publicly available datasets, we did not use any other versions of the M&Ms dataset during training. Regarding the issue where the unlabeled domain seems not to be learned, you can log the quality of the pseudo-labels for the unlabeled samples in each domain in real-time. This would allow you to observe whether there is consistently no learning (including unlabeled samples from the same domain as the labeled ones) or if overfitting occurs in the later stages of training. Dice and DC are the same metric; Dice is defined separately, while DC is calculated using the medpy package. You can simply refer to the DC results.

Thank you very much for your reply, at the moment it feels like there is a problem with the data enhancement part, but the data enhancement calls a function inside the environment. I would like to ask you the version of python, torch and cuda you were training on at the time

**Comment by TalibDaryabi** [comment]
@SilverBach805  were you able to manage and download the M&Ms dataset?

**Comment by koi805** [comment]
> [@SilverBach805](https://github.com/SilverBach805) were you able to manage and download the M&Ms dataset?

Yeah, I have downloaded using the link provided by the author
